ocs_ci/ocs/resources/csi_addons.py

```
@@ -231,3 +231,42 @@ def get_csi_addons_config_value(key: str, default: str = "") -> str:
+        constants.CSI_CEPHFS_ADDON_NODEPLUGIN_LABEL_420
+        if driver == "cephfs"
+        else constants.CSI_RBD_ADDON_NODEPLUGIN_LABEL_420
+    )
+    namespace = config.ENV_DATA.get("cluster_namespace", "openshift-storage")
+    pod_ocp = OCP(kind="pod", namespace=namespace)
+    pods = pod_ocp.get(
+        selector=label,
+        field_selector=f"spec.nodeName={node_name}",
+    )["items"]
+    logger.assertion(
+        f"csi-addons pod with label {label} " f"exists on node {node_name}"
```

Ruff (0.16.1)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)